src/pages/home/report/ReportActionCompose.js

```diff
@@ -1185,9 +1209,12 @@ function ReportActionCompose({
                                         style={[styles.textInputCompose, isComposerFullSize ? styles.textInputFullCompose : styles.flex4]}
                                         maxLines={maxComposerLines}
                                         onFocus={() => setIsFocused(true)}
-                                        onBlur={() => {
+                                        onBlur={(e) => {
                                             setIsFocused(false);
                                             resetSuggestions();
+                                            if (e.relatedTarget && e.relatedTarget === actionButtonRef.current) {
+                                                isKeyboardVisibleWhenShowingModalRef.current = true;
```

Comment:
This caused a regression https://github.com/Expensify/App/issues/26489. After setting this value to true we had to reset it to false as soon as a menu item is selected otherwise we will focus the composer for a brief period while opening the RHP.